docs/data/routes/docs/nextjs/experience-editor/walkthrough/en.md

```diff
@@ -3,4 +3,109 @@ name: walkthrough
 routeTemplate: ./data/component-templates/article.yml
 title: Connect to the Experience Editor
 ---
-# Walkthrough: Connect to the Experience Editor
\ No newline at end of file
+# Walkthrough: Connect your Next.js app to the Experience Editor
+
+A Next.js application utilizes the [JSS Http rendering engine](/docs/fundamentals/services/view-engine) feature for integration to the Sitecore Experience Editor. This article will guide you through the required configuration.
```

Comment:
Can we deep link to that header in the view engine page? Not sure our docs system allows.

Reply:
Agree, wanted a deep link here. Didn't think we could but at second inspection, it appears we can: https://jss.sitecore.com/docs/fundamentals/services/view-engine#http-rendering-engine
I'll update.